sql/core/src/main/scala/org/apache/spark/sql/internal/SessionState.scala

```diff
@@ -171,6 +171,7 @@ private[sql] class SessionState(sparkSession: SparkSession) {
   }
 
   def addJar(path: String): Unit = {
```

Comment:
so the `addJar` is not session-scoped by definition? I think `sparkSession.sparkContext.addJar(path)` is also cross-session